corehq/apps/export/management/commands/download_saved_export.py

```diff
@@ -0,0 +1,45 @@
+import shutil
+from datetime import datetime
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from corehq.apps.export.dbaccessors import get_properly_wrapped_export_instance
+from corehq.util.files import safe_filename
+
+
+def download_saved_export(export_id, dest_dir=None):
+    # Downloads the latest saved export to shared-directory
+    dest_dir = (dest_dir or settings.SHARED_DRIVE_ROOT).rstrip()
```

Comment:
You don't need the or because you've already specified a defualt in the argument